pages/contracts.tsx

```diff
@@ -111,6 +111,7 @@ const ContractList = () => {
             <Table>
               <thead style={{ textTransform: 'capitalize', fontSize: isMobile ? 12 : 14 }}>
                 <tr>
+                  <th style={{ width: '1rem' }}>{t(`#`)}</th>
```

Comment:
Why fix the width of this column, and how would it be if `number` is great that width of `number` is wider than `1rem`

Reply:
If we didn't give a width, this width may not be uniform for each page.
I'll remove it, first.

Reply:
A `min-width` could help various pages to have the same width while keep it flexible for large number which requires more width